sdcm/cluster.py

```diff
@@ -1470,16 +1472,22 @@ def copy_scylla_debug_info(self, node, scylla_debug_file):
         :returns: path on monitor node
         :rtype: {str}
         """
-        if not os.path.exists(os.path.join(node.logdir, os.path.basename(scylla_debug_file))):
-            node.remoter.receive_files(scylla_debug_file, node.logdir)
+        base_scylla_debug_file = os.path.basename(debug_file)
+        transit_scylla_debug_file = os.path.join(node.parent_cluster.logdir,
```

Comment:
We are in Python 3!!! **next time** use pathlib.Path

Reply:
Think about it , but think it could be ported to branch 3.2, because wasn't sure was decode backtrace ported there or not